app/templates/identity/api.hbs

```diff
@@ -0,0 +1,153 @@
+		<li><a href="https://github.com/Wikia/mercury/tree/dev/front/main/app/templates/components/design-system" target="_blank">Global Navigation in Mercury (HTMLBars, Ember)</a></li>
+		<li><a href="https://github.com/Wikia/app/tree/dev/extensions/wikia/DesignSystem/services">Oasis (MediaWiki, PHP)</a></li>
+		<li><a href="https://github.com/Wikia/upstream/tree/dev/wp-content/themes/upstream/modules/design-system" target="_blank">Upstream (Wordpress, PHP)</a></li>
+	</ul>
+
+	<p>
+		Logic for the Global Footer is straightforward and doesn't require much explanation.
+		Logic for the Global Navigation is more complex, here are highlights:
+	</p>
+	<ul>
+		<li>
+			For each section with <code class="format-code">links</code> and <code class="format-code">header</code>
+			properties we display a dropdown where <code class="format-code">header</code> maps to the dropdown entry-point
+			and <code class="format-code">links</code> to the list of links which should be rendered as the dropdown content.
+		</li>
+		<li>
+			For each section with <code class="format-code">links</code> but without <code class="format-code">header</code>
+			properties we display links without using a dropdown.
+		</li>
+		<li>
+			Section <code class="format-code">fandom_overview</code> is returned only for the languages
+			that have hubs implemented (e.g. <a href="http://fandom.wikia.com/games" target="_blank">http://fandom.wikia.com/games</a> for English).
+		</li>
+		<li>
+			Section <code class="format-code">search</code> has a property <code class="format-code">module</code> with properties:
+			<ul>
+				<li>
+					<code class="format-code">results</code> — URL for the search results page and the name of the search query parameter.
+				</li>
+				<li>
+					<code class="format-code">placeholder-active</code> and <code class="format-code">placeholder-inactive</code>
+					 — search input placeholders which should be switched between when user focuses / blurs the input.
+				</li>
+				<li>
+					<code class="format-code">suggestions</code> (optional) — URL for the suggestions API and the name
+					of the search query parameter. If this property is not present then the search suggestions should not be initialized.
+				</li>
+			</ul>
+		</li>
+		<li>
+			API returns either <code class="format-code">user</code> or <code class="format-code">anon</code> section,
+			based on the authentication header. Both return a header for the dropdown and a list of links.
+		</li>
+		<li>
+			Part of the links for the user / anon menu are of <code class="format-code">link_authentication</code> type
+			which comes with an additional property — <code class="format-code">param-name</code>.
+			It can be used in the link's URL to make sure that the user is properly redirected after doing an authentication action.
```

Comment:
There is a link to the example response in section "Fetch the data". I wouldn't want us to inline a full response in RP because it would be hard to maintain.